packages/react-vapor/src/components/modalWizard/ModalWizard.tsx

```diff
@@ -12,7 +12,7 @@ import {StepProgressBar} from '../stepProgressBar';
 export interface ModalWizardProps
     extends Omit<IModalCompositeOwnProps, 'modalBodyChildren' | 'validateShouldNavigate'> {
     id: string;
-    onFinish?: () => unknown;
+    onFinish?: (close: () => void) => void;
```

Comment:
onFinish could genuinely return any type and we don't really care, so there is no point of enforcing it to be void.

```suggestion
    onFinish?: (close: () => void) => unknown;
```